charts/lighthouse/Makefile

```diff
@@ -1,12 +1,18 @@
 CHART_REPO := http://jenkins-x-chartmuseum:8080
 CURRENT=$(pwd)
+GO_NOMOD := GO111MODULE=off go
 NAME := lighthouse
 OS := $(shell uname)
 
-build: clean
+HELMDOCS := $(GOPATH)/bin/helm-docs
+$(HELMDOCS):
+	$(GO_NOMOD) get -u github.com/norwoodj/helm-docs/cmd/helm-docs
```

Comment:
Maybe something as dirty as adding a `git checkout -- ../../go.mod ../../go.sum` and changing this to `GO111MODULE=on go get ...` is the answer. That would fetch/build correctly and undo changes to `go.mod` and `go.sum` that would otherwise be annoying...hrrrrrrrm.

Reply:
trying to `cd /tmp` temporarily...

Reply:
...that would mess things up if there were local changes to `go.mod` and `go.sum`, though. Augh.

Reply:
```go
GO_MOD := GO111MODULE=on go
NAME := lighthouse	NAME := lighthouse
OS := $(shell uname)	OS := $(shell uname)


build: clean	HELMDOCS := $(GOPATH)/bin/helm-docs
$(HELMDOCS):
	cd /tmp
	$(GO_MOD) get -u github.com/norwoodj/helm-docs/cmd/helm-docs
	cd -
```

Shouldn't that do it ?

Reply:
Aaaah, yeah, that'll do for now at least. We may want to end up changing to downloading a release tarball for the appropriate architecture from https://github.com/norwoodj/helm-docs/releases, exploding it, running it, and then deleting it, but the `cd /tmp` should help for now. I'm also changing #915 to do the helm steps before calling the root level `make verify-code-unchanged`...

Reply:
is it possible to use `docker run ...` ? if so i can run it in a container instead of trying to install it

Reply:
I...honestly don't know. =)